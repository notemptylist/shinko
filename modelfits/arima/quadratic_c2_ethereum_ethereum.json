{
    "numlags": 400,
    "results": [
        {
            "mean": -0.0004212428470744758,
            "order": [
                3,
                1,
                3
            ],
            "rmse": 3.632184596575837,
            "tstamp": 1640052627.6560237
        },
        {
            "mean": -0.0004212428470744758,
            "order": [
                3,
                1,
                1
            ],
            "rmse": 3.943575108991711,
            "tstamp": 1640052627.6560237
        },
        {
            "mean": -0.0004212428470744758,
            "order": [
                1,
                1,
                1
            ],
            "rmse": 4.135434099396287,
            "tstamp": 1640052627.6560237
        }
    ],
    "stream": "quadratic_c2_ethereum_ethereum.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            0,
            3
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            3
        ],
        [
            0,
            3,
            0
        ],
        [
            0,
            3,
            1
        ],
        [
            0,
            3,
            3
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            3
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            3
        ],
        [
            1,
            3,
            0
        ],
        [
            1,
            3,
            1
        ],
        [
            1,
            3,
            3
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            3
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            3
        ],
        [
            2,
            3,
            0
        ],
        [
            2,
            3,
            1
        ],
        [
            2,
            3,
            3
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            3
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            3,
            0
        ],
        [
            3,
            3,
            1
        ],
        [
            3,
            3,
            3
        ],
        [
            4,
            0,
            0
        ],
        [
            4,
            0,
            1
        ],
        [
            4,
            0,
            3
        ],
        [
            4,
            1,
            0
        ],
        [
            4,
            1,
            1
        ],
        [
            4,
            1,
            3
        ],
        [
            4,
            3,
            0
        ],
        [
            4,
            3,
            1
        ],
        [
            4,
            3,
            3
        ],
        [
            5,
            0,
            0
        ],
        [
            5,
            0,
            1
        ],
        [
            5,
            0,
            3
        ],
        [
            5,
            1,
            0
        ],
        [
            5,
            1,
            1
        ],
        [
            5,
            1,
            3
        ],
        [
            5,
            3,
            0
        ],
        [
            5,
            3,
            1
        ],
        [
            5,
            3,
            3
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            3
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            3
        ],
        [
            6,
            3,
            0
        ],
        [
            6,
            3,
            1
        ],
        [
            6,
            3,
            3
        ]
    ],
    "version": "0.1"
}