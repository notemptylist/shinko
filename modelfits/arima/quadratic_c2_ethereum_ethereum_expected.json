{
    "numlags": 400,
    "results": [
        {
            "mean": 0.5186950137643285,
            "order": [
                1,
                1,
                1
            ],
            "rmse": 1.8501358551841591,
            "tstamp": 1639952428.61274
        },
        {
            "mean": 0.5186950137643285,
            "order": [
                3,
                1,
                1
            ],
            "rmse": 1.8509473211276055,
            "tstamp": 1639952428.61274
        },
        {
            "mean": 0.5186950137643285,
            "order": [
                0,
                0,
                2
            ],
            "rmse": 1.8526402139659888,
            "tstamp": 1639952428.61274
        }
    ],
    "stream": "quadratic_c2_ethereum_ethereum_expected.json",
    "todo": [
        [
            0,
            0,
            0
        ],
        [
            0,
            0,
            1
        ],
        [
            0,
            1,
            0
        ],
        [
            0,
            1,
            1
        ],
        [
            0,
            1,
            2
        ],
        [
            0,
            2,
            0
        ],
        [
            0,
            2,
            1
        ],
        [
            0,
            2,
            2
        ],
        [
            1,
            0,
            0
        ],
        [
            1,
            0,
            1
        ],
        [
            1,
            0,
            2
        ],
        [
            1,
            1,
            0
        ],
        [
            1,
            1,
            2
        ],
        [
            1,
            2,
            0
        ],
        [
            1,
            2,
            1
        ],
        [
            1,
            2,
            2
        ],
        [
            2,
            0,
            0
        ],
        [
            2,
            0,
            1
        ],
        [
            2,
            0,
            2
        ],
        [
            2,
            1,
            0
        ],
        [
            2,
            1,
            1
        ],
        [
            2,
            1,
            2
        ],
        [
            2,
            2,
            0
        ],
        [
            2,
            2,
            1
        ],
        [
            2,
            2,
            2
        ],
        [
            3,
            0,
            0
        ],
        [
            3,
            0,
            1
        ],
        [
            3,
            0,
            2
        ],
        [
            3,
            1,
            0
        ],
        [
            3,
            1,
            2
        ],
        [
            3,
            2,
            0
        ],
        [
            3,
            2,
            1
        ],
        [
            3,
            2,
            2
        ],
        [
            6,
            0,
            0
        ],
        [
            6,
            0,
            1
        ],
        [
            6,
            0,
            2
        ],
        [
            6,
            1,
            0
        ],
        [
            6,
            1,
            1
        ],
        [
            6,
            1,
            2
        ],
        [
            6,
            2,
            0
        ],
        [
            6,
            2,
            1
        ],
        [
            6,
            2,
            2
        ]
    ],
    "version": "0.1"
}